home-assistant-plugin/custom_components/xknx/const.py

```diff
@@ -26,6 +26,8 @@
 CONF_RESET_AFTER = "reset_after"
 
 ATTR_COUNTER = "counter"
+ATTR_LAST_TELEGRAM_IA = "last_telegram_IA"
```

Comment:
in `knx_event` we use "source". I think it would make sense to use similar names.